README.md

```diff
@@ -3,5 +3,68 @@
 
 ## Testing
 
-Coverage report is available from
+### Preparation
+1. Clone the following repositories:
+    * https://github.com/osbuild/image-builder-frontend
```

Comment:
I think we can leave the image-builder-frontend and insights-proxy setup out of the image-builder readme.